flashinfer/gemm/kernels/cutile/ragged_block_scaled_bmm_cutile.py

```
@@ -0,0 +1,613 @@
+    `m_indptr` segment offset; it bounds the largest BLOCK_M the kernel may select
+    (BLOCK_M must divide it) because the kernel indexes rows as `m_start // BLOCK_M`.
+    Default 128. Pass 256 (with 256-aligned segments) to enable the large-M
+    BLOCK_M=256 fast path. This cannot be validated at runtime without a host sync
+    that would break CUDA-graph capture, so it is a caller contract.
+
+    `out` is an optional pre-allocated ``(total_m, N)`` output tensor written in
+    place (must be contiguous and match the output dtype). When None, an output is
+    allocated and returned. Passing `out` lets callers avoid an extra copy.
+    """
+    # Validate inputs
+    assert transpose_a == False and transpose_b == True, "Only NT layout is supported"
```

Ruff (0.15.21)

[error] Avoid equality comparisons to `False`; use `not transpose_a:` for false checks

Replace with `not transpose_a`

(E712)